app/serializers/event.js

```diff
@@ -13,9 +13,15 @@ export default ApplicationSerializer.extend(CustomPrimaryKeyMixin, {
 
   serialize() {
     const json = this._super(...arguments);
-    delete json.data.relationships['general-statistics'];
-    delete json.data.relationships['order-statistics'];
-
+    try {
+    delete json.data.relationships['general-statistics'];		
+     delete json.data.relationships['order-statistics'];
+ } catch(e){
+ 	console.error(e);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mixed spaces and tabs.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4983023)